### PR TITLE
Node 1228 skill map form add comment field

### PR DESCRIPTION
**Pull request by chadiegilnerubia**
Ticket ID:
Skill Map Form - Comments field
[#1228](https://github.com/nerubia/kh360-react/issues/1228)

### CODE REVIEW

File: src/services/user-service.ts [hunk 1]
```diff
@@ -959,6 +964,7 @@ export const getSkillMapRatings = async (skill_map_administration_id: number, us
     user_skill_map_ratings: userSkillMapRatings,
     skill_map_administration: skillMapAdministration,
     skill_map_result_status: skillMapResult.status,
+    comments: comment?.comments,
```
Comment (jatnerubia) [inline src/services/user-service.ts:967]: I think you can use `skillMapResult.comments` here

**Review by jatnerubia — CHANGES_REQUESTED** [review]
just a minor change ^^

File: src/services/user-service.ts (hunk 1 above)
Comment (chadiegilnerubia) [reply src/services/user-service.ts]: done :3

**Review by jatnerubia — APPROVED** [review]
Cool

**Pull request merged**